### PR TITLE
updated db schema to include additional info field and many-one arrows

### PR DESCRIPTION
**Pull request by DavidSciola**

### CODE REVIEW

File: documentation/db_schema.PNG
Comment (kevinhquach) [inline documentation/db_schema.PNG:1]: Not sure why we're deleting this.

**Review by kevinhquach — CHANGES_REQUESTED** [review]
Good new schema, not sure why we're removing the old one.

File: documentation/db_schema.PNG
Comment (DavidSciola) [reply documentation/db_schema.PNG:1]: You're right, ill re-add the old schema

**Review by DavidHaighton — APPROVED** [review]
NICE!!!

**Comment by DavidHaighton** [comment]
> Good new schema, not sure why we're removing the old one.

I think it's alright because we don't need the old one. it's still under the version 1 release tag so i think its good

**Review by kevinhquach — APPROVED** [review]

**Pull request merged**